Kickstarter-iOS/Views/Controllers/ProjectNavigatorViewController.swift

```diff
@@ -4,6 +4,8 @@ import Prelude
 import UIKit
 
 internal protocol ProjectNavigatorDelegate: class {
+  /// Call when a page view controller has completed transitioning.
```

Comment:
that makes sense. i think 90% of our delegate comments are "Call when" so we should consider updating across the board.